CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+#Contributing to Friendly Plans.
+##About
+Friendly Plans is an application supporting autism treatment.
+It provides digital version of activity schedules. More about treatment (ENG):
+  
+- http://www.pcdi.org/resources/videos.html
+
+The app is being used and able to download from google play store:
```

Comment:
The app is being used by group of users and therapists. Everyone can download it from google play store: